src/python/pants/notes/2.0.x.rst

```diff
@@ -5,6 +5,108 @@ This document describes releases leading up to the ``2.0.x`` ``stable`` series.
 
 See https://www.pantsbuild.org/v2.0/docs/release-notes-2-0 for an overview of the changes in this release.
 
+2.0.0b1 (9/23/2020)
+-------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Default to setting `LC_ALL` and `LANG` in subprocesses again (#10800)
+  `PR #10800 <https://github.com/pantsbuild/pants/pull/10800>`_
+
+* Default to `[python-infer].inits = false` (#10813)
+  `PR #10813 <https://github.com/pantsbuild/pants/pull/10813>`_
+
+* Deprecate `--interpreter-constraints` option where it no-ops (#10833)
+  `PR #10833 <https://github.com/pantsbuild/pants/pull/10833>`_
+
+* Upgrade setuptools from 49.6.0 to 50.3.0 and wheel from 0.31.1 to 0.35.1 (#10803)
+  `PR #10803 <https://github.com/pantsbuild/pants/pull/10803>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Restore support for third-party requirements with MyPy (#10811)
+  `PR #10811 <https://github.com/pantsbuild/pants/pull/10811>`_
```

Comment:
Is this actually "new"?

Reply:
Yeah, we never released it to end users. In b0, we added but reverted before it ever hit users.